CRAN_questions.md

```diff
@@ -0,0 +1,19 @@
+This document briefly presents some questions this working group has discussed about CRAN to help it being successful.
+
+ * CRAN does an amazing amount of work. What are the most critical issues that CRAN is facing?
+ * CRAN has worked relatively similar for the first 20 years. What is the future of CRAN for the next 20 years?
+ * There is some tension between packages' developers and users' interests respect to checks. What is the purpose of CRAN checks? 
```

Comment:
Perhaps we could focus this more on the need for transparency, e.g., "how can we help developers be better informed about CRAN checks, so as to reduce the friction in CRAN/developer interactions?"

Reply:
I like your question (I added it).
We discussed about the purpose of CRAN (or other repositories) checks: To detect/improve packages vs to protect users (but they are not exclusive). I think depending from how we see them emerges the friction between CRAN and developers. Anyway, this question might lead to understand their viewpoint

Reply:
What do you think about generalizing it to something like "how can we help reduce the friction in CRAN/developer interactions?"

Reply:
Sorry Will, I forgot about your suggestion (I'm not sure I received a notification, was reminded by the other PR notification #13 ). I agree with your wording, I think it is better to have more general questions and see where they lead.  Thanks!